### PR TITLE
HDDS-5610. Remove or clarify doc regarding OM federation

### PR DESCRIPTION
**Pull request by sky76093016**
## What changes were proposed in this pull request?

In the [OM](https://github.com/apache/ozone/blob/1a304ba81c9d52e2fb3a67e669d08c729a4113ae/hadoop-hdds/docs/content/concept/OzoneManager.md) part of the Ozone document, OM federation is not implemented, but there is no remarks, which will cause users to mislead when reading.

**Solution: clarify that this is the possibility of future improvement by adding a note after this paragraph.**

**If OM federation is successfully implemented in the future, delete this sentence.**

## What is the link to the Apache JIRA

https://issues.apache.org/jira/projects/HDDS/issues/HDDS-5610

## How was this patch tested?

Local hugo pass.

image

image

### CODE REVIEW

File: hadoop-hdds/docs/content/concept/OzoneManager.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This is very different from HDFS which provides a single rooted file system.
 
 Ozone's namespace is a collection of volumes or is a forest instead of a
 single rooted tree as in HDFS. This property makes it easy to deploy multiple
-OMs for scaling.
+OMs for scaling(the future development direction has not yet been realized).
```
Comment (jojochuang) [inline hadoop-hdds/docs/content/concept/OzoneManager.md:44]: more appropriate English would be:

`(pending future development)`

Comment (sky76093016) [reply hadoop-hdds/docs/content/concept/OzoneManager.md:44]: Thanks @jojochuang for reviewing this.This seems to be a better choice.

**Review by kerneltime — APPROVED** [review]

**Review by cku328 — CHANGES_REQUESTED** [review]
Thanks @sky76093016 for updating this docs.
In Chinese part, I think it should be `发展` -> `开发`.

**Comment by sky76093016** [comment]
> Thanks @sky76093016 for updating this docs.
> In Chinese part, I think it should be `发展` -> `开发`.

Thanks @cku328  for reviewing it.Chinese part has been modified.

**Review by cku328 — APPROVED** [review]
Overall LGTM.

**Review by bharatviswa504 — APPROVED** [review]

**Comment by bharatviswa504** [comment]
Thank You @sky76093016 for fix and @cku328 for review.

**Pull request merged**